### PR TITLE
Add home page and blog

### PR DESCRIPTION
**Pull request by sGerli**
Includes:
- New home page
- New blog page with pagination
- Moved css to scss
- Added excerpts to the last posts
- Fixed navigation wrapping issues

Supersedes #57 and #58

### CODE REVIEW

File: _data/home.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+heading: Bring life back to your Pebble
+about: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse vehicula, purus vitae vulputate varius, ipsum orci maximus erat, nec tempor massa purus id purus. Nullam ligula augue, ultricies id sollicitudin quis, dictum sit amet leo.
```
Comment (sGerli) [inline _data/home.yml]: Dummy text still needs to be changed.

Comment (ishotjr) [inline _data/home.yml]:
```suggestion
about: On December 7th, 2016, <a href="https://web.archive.org/web/20180705003113/https://blog.getpebble.com/2016/12/07/fitbit/">Pebble announced that they were ceasing operations</a>.  Two days later, on December 9th, 2016, <a href="https://rebble.io/2016/12/09/rebble-pebble-reborn.html">the Rebble web site went live</a>, announcing our intention to "maintain and advance Pebble functionality, in the absence of Pebble Technology Corp."  We have fulfilled much of this goal via <a href="http://rebble.io/2018/07/01/rebble-era.html">Rebble Web Services</a>, which restores most of the functionality that the Pebble community knows and loves.  Read about our journey in the <a href="/blog/">blog</a>, find detailed instructions in our <a href="/howto/">HOWTO</a>, or simply...
```

(I might revise this later, but wanted to get you something ASAP so we can ship your amazing work!! ❣️ )

File: blog/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
```
[the remark this answers is not in the document]
Comment (sGerli) [reply blog/index.html]: That's weird, it's working for me.

image

[the remark this answers is not in the document]
Comment (sGerli) [reply blog/index.html]: @ishotjr can you check that you're on the right branch?

[the remark this answers is not in the document]
Comment (ishotjr) [reply blog/index.html]: @sGerli ah - totally failed to notice we'd moved to `home-plus-blog` - thanks!! 🤦‍♂

**Comment by ishotjr** [comment]
@sGerli FYI had to force CSS to refresh for `summary` styling to display correctly in my browser ... 🤔 ❤️

**Comment by sGerli** [comment]
Yeah, I noticed that. To force all browsers to update we would have needed to modify the css import with a query string like `?version=1.2.0`.

**Comment by ishotjr** [comment]
@sGerli yep, let's version it in future! 👍

**Comment by sGerli** [comment]
In other news: This PR fixes the navbar urls so now you should be able to tick the `Enforce HTTPS` checkbox in the settings to fully move to https.

**Pull request merged**